### ISSUE TITLE
The ability to chose wich provider is first

### ISSUE THREAD

**Issue by castilio20** [PrismLauncher/PrismLauncher#2448]
### Role

i play modded mc

### Suggestion

i want the ability to chose what provider it first i am using more cf that modrinth and it kinda anoyng to have md first

### Benefit

benefit would be so you dont need to switch evry time providers

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by KujouShiori** [comment]
默认下载源太慢了，才几KB

**Comment by Marks12520** [comment]
Yeah that was something useful that other launchers like ATLauncher have

**Comment by KujouShiori** [comment]
> 是的，这是其他启动器（如 ATLauncher）所具有的有用之处

搭配梯子时从modrinth的下载速度还不错，但总挂梯子也不是个事
